pkg/process/metadata/workloadmeta/extractor.go

```diff
@@ -100,3 +133,23 @@ func Enabled(ddconfig config.ConfigReader) bool {
 func hashProcess(pid int32, createTime int64) string {
 	return "pid:" + strconv.Itoa(int(pid)) + "|createTime:" + strconv.Itoa(int(createTime))
 }
+
+// GetAllProcessEntities returns all processes Entities stored in the WorkloadMetaExtractor cache and the version
+// of the cache at the moment of the read
+func (w *WorkloadMetaExtractor) GetAllProcessEntities() (map[string]*ProcessEntity, int32) {
```

Comment:
Not an issue introduced by this PR specifically, but I think it would be worth adding some logs that indicate the size of the process entity cache, to help with debugging in case we see reports of increased memory usage on hosts with lots of processes.